spec/controllers/signatures_controller_spec.rb

```diff
@@ -160,6 +160,11 @@ def do_post(options = {})
           expect(assigns(:signature).petition).to eq(petition)
         end
 
+        it "has not changed the default TRUE value for notify_by_email" do
```

Comment:
I think it's reasonable.  Here using ``eq true`` feels ok because we actually do want the value to be literally ``true`` or literally ``false``.